odoo/addons/base/models/ir_model.py

```diff
@@ -1408,10 +1409,16 @@ def _reflect_inherits(self, model_names):
                 ] + [
                     (model_id, get_model_id(parent_name), get_field_id(field))
                     for parent_name, field in cls._inherits.items()
+                ] + [
+                    (model_id, get_model_id(field.comodel_name), get_field_id(field_name))
+                    for (field_name, field) in inspect.getmembers(cls)
+                    if isinstance(field, fields.Field)
+                    if field.type == 'many2one' and not field.related and field.delegate
```

Comment:
Small question: Is related delegate field possible and does it make sense? @rco-odoo